notebooks/Specviz2dExample.ipynb

```diff
@@ -0,0 +1,117 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "73ca6a10",
+   "metadata": {},
+   "source": [
+    "# Specviz2d Demonstration Notebook\n",
+    "This notebook demonstrates the Specviz2d API in the Notebook setting. General documentation about Jdaviz UI and interactions can be found here: https://jdaviz.readthedocs.io/en/latest/index.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec894f29",
+   "metadata": {},
+   "source": [
+    "## Create Specviz2d via Helper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f99e3d2d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jdaviz import Specviz2d\n",
+    "specviz2d = Specviz2d()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ed07713",
+   "metadata": {},
+   "source": [
+    "## Display Specviz2d"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45802673",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz2d.app"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c5d391ee",
+   "metadata": {},
+   "source": [
+    "## Load a File"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a60eaa88",
+   "metadata": {},
+   "source": [
+    "Download data from: https://stsci.app.box.com/s/exnkul627fcuhy5akf2gswytud5tazmw\n",
+    "\n",
+    "Set the path to the data in the following cell"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b962e74a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = \"/path/to/det_image_seq5_MIRIMAGE_P750Lexp1_s2d.fits\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ec94837b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz2d.load_data(filename)"
+   ]
+  },
```

Comment:
Can we do this programmatically using `astropy.utils.data.download_file` as in other notebooks?  I tried swapping it out, but it seems to not recognize the file format and unlike `specviz.load_spectrum`, `specviz2d.load_data` does not seem to accept the format kwarg - so perhaps this is out of scope?

Reply:
I think I know why... The link brings you to the *Box page* where the data is, but it is not a direct link to the file itself. If @javerbukh fixed that Box link to be direct link to file, direct download will work.

Reply:
Also, I accidentally found a bug. I will open a PR to this PR.